src/Compilers/VisualBasic/Portable/Lowering/UseTwiceRewriter.vb

```diff
@@ -1,5 +1,7 @@
 ﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#Const DONT_USE_BYREF_LOCALS_FOR_USE_TWICE = True
```

Comment:
I believe this affects more scenarios than we would like to. We only need to change behavior of UseTwiceArrayAccess method.

Reply:
Updated.